storage/acceptance/aws_s3_mount_test.go

```diff
@@ -0,0 +1,74 @@
+package acceptance
+
+import (
+	"fmt"
+	"os"
+	"testing"
+
+	"github.com/databrickslabs/databricks-terraform/common"
+	"github.com/databrickslabs/databricks-terraform/compute"
+	"github.com/databrickslabs/databricks-terraform/internal/acceptance"
+	. "github.com/databrickslabs/databricks-terraform/storage"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"github.com/stretchr/testify/assert"
+
+	"github.com/databrickslabs/databricks-terraform/internal/qa"
+)
+
+func getAwsS3MountConfig(t *testing.T, mountName string) string {
+	if os.Getenv("TEST_AWS_MOUNT_CLUSTER_ID") != "" {
```

Comment:
1) add in `otherVars` change from https://github.com/databrickslabs/terraform-provider-databricks/blob/6290ae42e6bd1c6e0152228208e8c9f58e55c5da/internal/qa/testing.go#L286 and remove `TEST_AWS_MOUNT_CLUSTER_ID`
2) `EnvironmentTemplate` is intended to be used unconditionally for test step configs - so that it's readable in single place what this step config does. if there is conditional - it should be two tests. so in this case it's two different tests.

Reply:
sure I will make a test that requires cluster id and another one that does not.